src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("./images/star.svg") no-repeat center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
move common part to the .stars-star selector

Reply:
done